about-page/about.html

```diff
@@ -0,0 +1,188 @@
+        <h3 class="hello">Hello Sharing World</h3>
+        <div class="topic">
+          <h1 class="topic1">UGANDA  ANNUAL CULTURAL FORUM</h1>
+        </div>
+        <div class="text">
+          <p class="text1">An Annual discourse, digest and celebration of the tourism and culture sectors of Uganda, the Pearl of Africa...A deep dive into our unique culture and beautiful nation!!!<span class="cut"> We will discuss and celebrate the rich culture of Uganda as a nation. Our key speakers will present and recieve feedback about their research into our culture. Dont miss entertainment from Ugandan cultural dancing groups as well.</span></p>
+        </div>
+        <div class="contact">
+          <p class="contact1">Please contact us on Email with any questions about the Uganda Annual Cultural Forum</p>
+          <p class="mail">ugandaculture@uganda.gov</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+<!-- ...//logo..// -->
+
+  <div class="logo">
+    <h2 class="logo-topic">Uganda Annnual forum Logo 2022</h2>
+    <div class="logo-texta">
+    <p class="logo-text">The Uganda Annual Culture Forum Logo was decided by competition from feb to March 2020<span class="cut1">After acception of design propossals from the public, a group of judges decided on the best design for the logo og Uganda Annual Cultral Forum. We believe in inclusion of the public in all decision making processes</span></p>
+      </div>
+    <div class="logo-img">
+      <img src="images/red.png" alt="" class="logo-img1">
+    </div>
+  </div>
+
+<!-- ....//previous forum..//   -->
+
+<section id="past-forum">
+<div class="past-texts">
+  <h2 class="past-topic">See the past Uganda Annual culture forums</h2>
+  <p class="past-text">Take a look at the past Uganda Annual culture forums that took place at Victoria Hotel and Pearl Hotel.</p>
+</div>
+
+<div class="heado">
+<div class="head">
+  <img src="images/people.jpg" alt="" class="past-img">
+  <div class="head-text">
+    <h1>2019</h1>
+    <h3>Culture forum at Victoria Hotel</h3>
+  </div>
+</div>
+
+<div class="head">
+  <img src="images/people2.jpg" alt="" class="past-img">
+  <div class="head-text">
+    <h1>2020</h1>
+    <h3>Culture forum at pritoria Hotel</h3>
+  </div>
+</div>
+</div>
+</section>
+
+<!-- ...///partners...// -->
+<section id="partners">
+<div class="partner-text">
+  <h2>Partners</h2>
+  <hr style="height:2px; width: 40px;border-width:0;color:#ff4500;background-color:#ff4500;">
+</div>
+<div class="partner-imgs">
+  <div class="partner-img">
+    <img src="images/logo1.svg" alt="">
+  </div>
+  <div class="partner-img">
+    <img src="images/logo2.svg" alt="">
+  </div>
+  <div class="partner-img">
+    <img src="images/logo3.svg" alt="">
+  </div>
+  <div class="partner-img">
+      <img src="images/logo4.svg" alt="">
+  </div>
+</div>
+
+</section>
+
+ <!-- ....//copyright.../ -->
+<div class="copy">
+  <p>@danielmatama copyright</p>
+</div>
+
+<script src="scripts.js"></script>
```

Comment:
[REQUIRED] Kindly ensure that your footers contain a logo on the left and text on the right. One of the aims of this project is to teach you to follow project requirements.

[REQUIRED] I noticed that you are not using semantic tags like section, article, and footer among others in a lot of places. Kindly fix this.
[REQUIRED]For desktop screens the about page should have a dark background while the homepage should have white bacckground